extensions/stackdriver/stackdriver.h

```diff
@@ -236,6 +242,7 @@ class StackdriverContext : public Context {
   bool is_tcp_;
   uint32_t context_id_;
   const bool is_initialized_;
+  absl::optional<bool> downstream_data_first_{};
```

Comment:
new to c++, why do we end the variable name with `_`?

Reply:
Not a must in C++ but a code style in Envoy. https://github.com/envoyproxy/envoy/blob/main/STYLE.md